### PR TITLE
[W5.4b][W14-A3]Wang Jinyi

### PR DESCRIPTION
**Pull request by WJY-norainu**
LO-ClassLevel

### CODE REVIEW

File: src/seedu/addressbook/data/person/Person.java [hunk 1]
```diff
@@ -10,10 +10,16 @@
  */
 public class Person implements ReadOnlyPerson {
 
+    /**
+     * A sequence number to keep track of sequence in which different persons are added
+     */
+    public static int nextSequenceNumber = 0;
```
Comment (kowshik-sundararajan) [inline src/seedu/addressbook/data/person/Person.java:16]: Should this be 0 or 1?

**Review by kowshik-sundararajan — COMMENTED** [review]
Try to maintain an imperative tone for commit messages: https://chris.beams.io/posts/git-commit/

Please close the PR after seeing the comments.

File: src/seedu/addressbook/data/person/Person.java (hunk 1 above)
Comment (WJY-norainu) [reply src/seedu/addressbook/data/person/Person.java:16]: Oh it's 1. As it is a 1-based index.
Thank you!

**Comment by kowshik-sundararajan** [comment]
Forgot to mention, please ensure that the build passes. I suspect the build failed because the tests have not been updated to account for `parseCommand()` accepting two parameters.

**Pull request closed without merging**